src/Doc/ThirdPartyLibraries.html.cmake

```diff
@@ -56,36 +60,33 @@ <h3><a name="_TocKDL"></a>KDL</h3>
 <p>License: LGPL version 2.1: <a href="#_LGPLv21">LGPL v2.1</a></p>
 
 
-<hr>
-
-<h3><a name="_TocLibArea"></a>libarea</h3>
-<p>Web site: <a href="https://github.com/danielfalck/libarea/">https://github.com/danielfalck/libarea/</a></p>
-<p>License: BSD-3-clause: <a href="https://github.com/danielfalck/libarea/blob/master/License.txt">https://github.com/danielfalck/libarea/blob/master/License.txt</a></p>
-
-
```

Comment:
looks like libarea was removed but AFAICT not mentioned anywhere else?

Reply:
According to slpitronics post in [this forum thread](https://forum.freecad.org/viewtopic.php?style=4&t=43857), we are using our own copy which is diverting from the upstream source. I can pack it back inside again, but it looks like the 3rd Party Library overviews on the web and everywhere else are a bit outdated. 
Should I add it back again (without version string) ?

Reply:
Unsure. Let's ping @sliptonic

Reply:
Let's add it back without version.